nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+
+    enable = mkEnableOption (lib.mdDoc "Enable the playerctld daemon.");
+
+    package = mkPackageOption pkgs "playerctl" {
+      default = [ "playerctl" ];
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = with pkgs; [ cfg.package ];
```

Comment:
`with pkgs;` is not needed in this context.

Reply:
corrected